boilerplate/app/navigation/primary-navigator.tsx.ejs

```diff
@@ -1,9 +1,17 @@
 import React from "react"
-import { createNativeStackNavigator } from "@react-navigation/native-stack"
+<% if(props.useExpo){ %>
+  import { createStackNavigator } from "@react-navigation/stack"
+<% } else{ %>  
+  import { createNativeStackNavigator } from "react-native-screens/native-stack"
+<% } %>
```

Comment:
Sorry I'm so picky about spacing, but we can't use Prettier on the template files:
- These imports shouldn't be indented; picture the generated file without the template conditionals, and you'll see that the imports should line up on the left.
- The if line needs more spacing: `if (props.useExpo) {`
- The else line needs a space too: `else {`
- If you add a `-` to the closing tag (eg `<% blah -%>`) the templater won't output the newline following the tag. Do this to all three template tags that form this conditional, and things should look right. (Generate a new Expo & non-Expo app to be sure!)
- The conditional below and the ones in `root-navigator.tsx.ejs` need the same fixes

Reply:
makes sense, thank you for catching these

Reply:
@bryanstearns Does the generated app get prettier-ed after generation? If so, would these things be fixed? (I'm not against fixing them here, just wondering.)

Reply:
Prettier doesn't fix extra single blank lines, but might fix indentation